resources/todomvc/vanilla-examples/javascript-web-components/src/styles/app.constructable.js

```diff
@@ -0,0 +1,5 @@
+const sheet = new CSSStyleSheet();
+sheet.replaceSync(
+    ":host {\n    display: block;\n\tbox-shadow: none !important;\n\tmin-height: 68px;\n}\n\n.app {\n\tbackground: #fff;\n\tmargin: 24px 16px 40px 16px;\n\tposition: relative;\n\tbox-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.2),\n\t            0 25px 50px 0 rgba(0, 0, 0, 0.1);\n}\n"
```

Comment:
I wonder why you didn't use template strings with "\`" so that it could be much more readable than with these `\n` everywhere? Is there a good reason?

Reply:
that's how my rollup plugin from https://github.com/WebKit/Speedometer/pull/254 generated the output. 
Can we modify there and update once that pr lands?

Reply:
updated in both prs - thanks @julienw !